<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QMainWindow" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>596</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sahel</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="accessibleName">
       <string>class2</string>
      </property>
      <property name="title">
       <string>  محدوده جست‌وجو  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="selectAllCheckBox">
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string>انتخاب همه   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="selectNoneCheckBox">
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string>هیچکدام   </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="font">
            <font>
             <family>Sahel</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="accessibleName">
            <string>class2</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelTotal">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="accessibleName">
              <string>classFDs</string>
             </property>
             <property name="text">
              <string>تعداد کل:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelSelectedCount">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="accessibleName">
              <string>classFDs</string>
             </property>
             <property name="text">
              <string>انتخاب‌شده:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxShowItems">
      <property name="accessibleName">
       <string>class2</string>
      </property>
      <property name="text">
       <string>نمایش موارد پیدا شده در حین جست‌وجو</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="labelSearchMethod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>متد جست‌وجو:       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMethod1">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>متد ۱ (سریع‌تر)   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMethod2">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>متد ۲ (منظم‌تر)   </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QCheckBox" name="checkBoxSkipDiacritics">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>نادیده گرفتن حرکت کلمات   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSkipCharTypes">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>نادیده گرفتن حالت‌های یک حرف       </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelPlus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>همه این کلمات:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelOr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>حداقل یکی از این کلمات:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelExact">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>عین این کلمه یا عبارت:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelOrder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>کلمات با رعایت ترتیب:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelNeg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>بدون این کلمات:   </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditPlus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditExact">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOrder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNeg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>class2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="labelStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>شروع با: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>    پایان با: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxGhafie">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>قافیه:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRadif">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>ردیف:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxCounter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>تعداد:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSingle">
          <property name="accessibleName">
           <string>class2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>جست‌وجو</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExamples">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>مثال‌ها</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>تایید</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>بستن</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditPlus</tabstop>
  <tabstop>lineEditOr</tabstop>
  <tabstop>lineEditExact</tabstop>
  <tabstop>lineEditOrder</tabstop>
  <tabstop>lineEditNeg</tabstop>
  <tabstop>lineEditStart</tabstop>
  <tabstop>lineEditEnd</tabstop>
  <tabstop>checkBoxGhafie</tabstop>
  <tabstop>checkBoxRadif</tabstop>
  <tabstop>checkBoxCounter</tabstop>
  <tabstop>lineEditSingle</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>btnExamples</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>selectAllCheckBox</tabstop>
  <tabstop>selectNoneCheckBox</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>checkBoxShowItems</tabstop>
  <tabstop>radioMethod1</tabstop>
  <tabstop>radioMethod2</tabstop>
  <tabstop>checkBoxSkipDiacritics</tabstop>
  <tabstop>checkBoxSkipCharTypes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
